test/versioned/esm-package/parse-json-instrumentation.mjs

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use-strict'
+
+// import util from 'util'
+
+export default function initialize(shim, parseJson) {
+  shim.wrap(parseJson, 'default', function wrappedParseJsonLib(_shim, orig) {
+    return function wrappedParseJsonFunc() {
+      const result = orig.apply(this, arguments)
+      result.isInstrumented = true
+      return result
+    }
+  })
+
+  shim.wrap(parseJson, 'JSONError', function wrappedParseJsonLib(_shim, orig) {
+    // const WrappedError = function wrappedError() {
```

Comment:
why is this commented out?

Reply:
@bizob2828 is helping me out with some debugging, so I pushed up what I was working on to make our debugging session faster/easier, will put this PR back into draft until said debugging is done